File: 1480.py
# Author: Zoljargal Gantumur
# Runtime: 36 ms, faster than 92.99% of Python3 online submissions
# Memory Usage: 14.1 MB, less than 33.33% of Python3 online submissions

class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        ans = []
        a = 0
        for i in nums:
            a+=i
            ans.append(a)
        return ans

File: 1480.cpp
/*
	Author: Zoljargal Gantumur
	Runtime: 4 ms, faster than 93.14% of C++ online submissions
	Memory Usage: 8.7 MB, less than 40.00% of C++ online submissions
*/

class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        vector<int> ans;
        int sum =0;
        for(int num: nums){
            sum+=num;
            ans.push_back(sum);
        }
        return ans;
    }
};
